voting-machine/host/src/lib.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright 2022 Risc0, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risc0_zkvm_host::{Prover, Receipt, Result};
+use risc0_zkvm_serde::{from_slice, to_vec};
+use std::fs;
+use tempfile::{NamedTempFile, TempPath};
+use voting_machine_core::{Ballot, SubmitBallotCommit, SubmitBallotParams, SubmitBallotResult};
+use voting_machine_core::{
+    FreezeVotingMachineCommit, FreezeVotingMachineParams, FreezeVotingMachineResult,
+};
+use voting_machine_core::{InitializeVotingMachineCommit, VotingMachineState};
+use voting_machine_methods::{FREEZE_ID, FREEZE_PATH, INIT_ID, INIT_PATH, SUBMIT_ID, SUBMIT_PATH};
```

Comment:
Two kinds of groupings for uses:

* by source
  * std, alloc, core
  * third party
  * current crate uses
* group together common prefixes

```suggestion
use std::fs;

use risc0_zkvm_host::{Prover, Receipt, Result};
use risc0_zkvm_serde::{from_slice, to_vec};
use tempfile::{NamedTempFile, TempPath};
use voting_machine_core::{Ballot, SubmitBallotCommit, SubmitBallotParams, SubmitBallotResult, FreezeVotingMachineCommit, FreezeVotingMachineParams, FreezeVotingMachineResult, InitializeVotingMachineCommit, VotingMachineState};
use voting_machine_methods::{FREEZE_ID, FREEZE_PATH, INIT_ID, INIT_PATH, SUBMIT_ID, SUBMIT_PATH};
```